sky/utils/schemas.py

```diff
@@ -57,6 +57,7 @@ def _check_not_both_fields_present(field1: str, field2: str):
             'required': [],
             'additionalProperties': False,
             'properties': {
+                # TODO(luca): update field to use minutes as well.
```

Comment:
```suggestion
                # TODO(luca): update field to use time units as well.
```
I assume this is what you meant